### ISSUE TITLE
Auto copy to clipboard

### ISSUE THREAD

**Issue by AdaptaRich** [thejeremybecker/ahk#2]
1) Is Ctrl+C really necessary? Please auto-execute or allow option to auto-execute Ctrl+C. 
2) Of course once auto-copied, do we really need to manually close it?

So, in summary Ctrl+Alt while dragging mouse and Ctrl+V to paste from clipboard. - Perfect

**Comment by thejeremybecker** [comment]
Good idea. Office OneNote already has this feature (Win + S key) but it was disabled in Win10 since Windows search now uses the OneNote key combo. Should be able to add similar functionality feature to screenclip as an additional function, where Ctrl + Alt + S copies screen area directly to clipboard without leaving a window open.
